### PR TITLE
Add new pipeline: MB'n'B

### PR DESCRIPTION
**Pull request by hawkrives**
This PR adds the pipeline Freedberg and @mollysrour were talking about yesterday, which runs two tree generation steps.

The first tree generation generates the consensus tree through MrBayes, which we feed into Ent and get out the initial list of non-monophyletic individuals.

The second tree generation removes those individuals from the data, then runs BEAST and JML on the reduced dataset.

We then present the JML distributions, probabilities, and results tables nicely in the UI, and we highlight any results with P<0.05 in the table in yellow.

Thoughts on the approach or the implementation welcome. I'll go over it at tomorrow's meeting, too.

(PS: this PR is actually only +314, if you discount the new tests.)

screen shot 2018-04-10 at 4 25 05 pm

screen shot 2018-04-10 at 4 25 07 pm

### CODE REVIEW

File: electron/run.js [hunk 1]
```diff
@@ -104,15 +116,25 @@ function onData(phase, data) {
...
 			.querySelector('#distributions')
 			.appendChild(makeTableFromObject(data.distributions))
+
 		document
 			.querySelector('#probabilities')
 			.appendChild(makeTableFromObject(data.probabilities))
-		document.querySelector('#results').innerHTML = `<pre>${ress}</pre>`
+
+		data.results = data.results.map(item => {
+			if (item.Probability < 0.05) {
```
Comment (rye) [inline electron/run.js]: Capital P?

Comment (rye) [reply electron/run.js]: Oh, JML output. NVM

Comment (hawkrives) [reply electron/run.js]: Yeah, that's what the column is called, so that's what we get as our object key.

**Comment by rye** [comment]
At Hawken's behest, I will merge this and make another release and inform the bio team yet again

**Comment by rye** [comment]
(once I get up to my room that is)

**Pull request merged**